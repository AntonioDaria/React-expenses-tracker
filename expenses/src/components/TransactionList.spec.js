import React from 'react';
import { render } from '@testing-library/react';
import TransactionList from './TransactionList';



describe('TransactionList component', () => {
    let container;

    beforeEach(() => {
      ({ container} = renderTransactionList());
    });

    it('should match the snapshot', () => {
        expect(container).toMatchSnapshot();
    });
});

const renderTransactionList= () => {
    return render(
      <TransactionList/>
    );
  };
  