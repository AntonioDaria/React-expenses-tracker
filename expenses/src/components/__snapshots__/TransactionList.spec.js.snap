// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionList component should match the snapshot 1`] = `
<div>
  <h3>
    History
  </h3>
  <ul
    class="list"
  >
    <li
      class="minus"
    >
      Flower
       
      <span>
        $
        -20
      </span>
      <button
        class="delete-btn"
      >
        x
      </button>
    </li>
    <li
      class="minus"
    >
      Salary
       
      <span>
        $
        300
      </span>
      <button
        class="delete-btn"
      >
        x
      </button>
    </li>
    <li
      class="minus"
    >
      Book
       
      <span>
        $
        -10
      </span>
      <button
        class="delete-btn"
      >
        x
      </button>
    </li>
    <li
      class="minus"
    >
      Camera
       
      <span>
        $
        150
      </span>
      <button
        class="delete-btn"
      >
        x
      </button>
    </li>
  </ul>
</div>
`;
